src/utils/__tests__/extractLinksRecursively.test.js

```diff
@@ -0,0 +1,15 @@
+const { extractLinksRecursively } = require('../extractLinksRecursively');
+
+describe('extractLinksRecursively', () => {
+  it('should return a unique set of links and anchor tags', () => {
+    expect(extractLinksRecursively('src/utils/__tests__/examples/duplicate-paths')).toMatchSnapshot();
+  });
+
+  it('should not include any external links', () => {
+    expect(extractLinksRecursively('src/utils/__tests__/examples/external-links')).toMatchSnapshot();
+  });
+
+  it('should not include anchor tags that do not go anywhere', () => {
```

Comment:
@AndyClifford I consider these links to be broken (either pointless or they could be a mistake). Probably another good requirement for a broken link detection tool

Suggestion: Fix these in a follow up if any are coming from the Astro site. Remove this test case when the Jekyll site is deleted

Reply:
Yeah looks like the Jekyll build automatically generates these random anchor tags e.g. `<a href="#" class="nav-list-expander">`. This isn't a problem with the astro site so can remove it after the migration